srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRuleFactory.java

```diff
@@ -0,0 +1,25 @@
+package gov.va.med.srcalc.web.view.admin;
+
+import gov.va.med.srcalc.service.AdminService;
+import gov.va.med.srcalc.service.InvalidIdentifierException;
+
+/**
+ * Constructs an appropriate instance of {@link EditExistingRule} to edit an
+ * existing rule.
+ */
+public class EditRuleFactory
```

Comment:
This factory is so simple. Do you really need it? Could you just move this code to `EditRuleController`?